Cartesian_kernel/include/CGAL/Cartesian/Point_3.h

```diff
@@ -45,6 +45,11 @@ class PointC3
   PointC3(const FT &x, const FT &y, const FT &z)
     : base(x, y, z) {}
 
+  PointC3(FT &&x, FT &&y, FT &&z)
+    : base(std::forward<FT>(x), std::forward<FT>(y), std::forward<FT>(z))
+  {}
```

Comment:
That code is not perfect forwarding, because `FT` is not a forwarding-reference.

Reply:
See https://en.cppreference.com/w/cpp/language/reference#Forwarding_references